heartbeat/IPaddr2

```diff
@@ -666,6 +676,34 @@ find_interface() {
 	return 0
 }
 
+#
+#        Kill dangling ip connections
+#
+kill_ip_connections() {
+	local ss_output
+
+	ipaddr="$1"
```

Comment:
This should be the first line, as it's the function variable(s), and make it local to avoid the possibility of it clashing with global variables.

Reply:
Ah yeah. It's not always ideal.

We dont want to change it there unless we're changing doing it as part of changing that code.